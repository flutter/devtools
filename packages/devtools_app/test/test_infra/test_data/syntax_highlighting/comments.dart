// Copyright 2022 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

/// Multiline dartdoc comment.
///
/// ```
/// doc
/// ```
///
/// ...
var a;

/*
 * Old-style dartdoc
 *
 * ...
 */
var b;

/* Inline block comment */
var c;

/**
 * Nested block
 *
 * /**
 *  * Nested block
 *  */
 */
var d;

/**
 * Nested
 *
 * /* Inline */
 */
var e;

/* Nested /* Inline */ */
var f;

// Simple comment
var g;

/// Dartdoc with reference to [a].
/// And a link to [example.org](http://example.org/).
var h;
