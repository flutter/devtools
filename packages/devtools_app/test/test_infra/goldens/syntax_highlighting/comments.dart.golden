>// Copyright 2022 The Chromium Authors. All rights reserved.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// Use of this source code is governed by a BSD-style license that can be
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// found in the LICENSE file.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>
>/// Multiline dartdoc comment.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
>///
#^^^ comment.block.documentation.dart
>/// ```
#^^^^^^^ comment.block.documentation.dart
>/// doc
#^^^ comment.block.documentation.dart
#   ^^^^ comment.block.documentation.dart variable.other.source.dart
>/// ```
#^^^ comment.block.documentation.dart
#   ^ comment.block.documentation.dart variable.other.source.dart
#    ^^^ comment.block.documentation.dart
>///
#^^^ comment.block.documentation.dart
>/// ...
#^^^^^^^ comment.block.documentation.dart
>var a;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/*
#^^ comment.block.dart
> * Old-style dartdoc
#^^^^^^^^^^^^^^^^^^^^ comment.block.dart
> *
#^^ comment.block.dart
> * ...
#^^^^^^ comment.block.dart
> */
#^^^ comment.block.dart
>var b;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/* Inline block comment */
#^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.dart
>var c;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/**
#^^^ comment.block.documentation.dart
> * Nested block
#^^^^^^^^^^^^^^^ comment.block.documentation.dart
> *
#^^ comment.block.documentation.dart
> * /**
#^^^^^^ comment.block.documentation.dart
> *  * Nested block
#^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
> *  */
#^^^^^^ comment.block.documentation.dart
> */
#^^^ comment.block.documentation.dart
>var d;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/**
#^^^ comment.block.documentation.dart
> * Nested
#^^^^^^^^^ comment.block.documentation.dart
> *
#^^ comment.block.documentation.dart
> * /* Inline */
#^^^ comment.block.documentation.dart
#   ^^^^^^^^^^^^ comment.block.documentation.dart comment.block.dart
> */
#^^^ comment.block.documentation.dart
>var e;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/* Nested /* Inline */ */
#^^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.dart
>var f;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>// Simple comment
#^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>var g;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
>
>/// Dartdoc with reference to [a].
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
#                              ^^^ comment.block.documentation.dart variable.name.source.dart
#                                 ^ comment.block.documentation.dart
>/// And a link to [example.org](http://example.org/).
#^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
#                  ^^^^^^^^^^^^^ comment.block.documentation.dart variable.name.source.dart
#                               ^^^^^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
>var h;
#^^^ storage.type.primitive.dart
#     ^ punctuation.terminator.dart
